identity-service/src/audiusLibsInstance.js

```diff
@@ -42,6 +42,20 @@ class AudiusLibsWrapper {
   getAudiusLibs () {
     return this.audiusLibsInstance
   }
+
+  async getAudiusLibsAsync () {
```

Comment:
why do we need this function? is it because of a server init timing issue with the requires?

Reply:
yeah-- we don't have any sort of method that will wait for libs init and it's nice to have. a better way of doing this would be via event-emitter, but i didn't want to introduce a whole pattern here for that

Reply:
Mind throwing in a comment here 🙂